Common/Inc/config.h

```diff
@@ -0,0 +1,58 @@
+#pragma once
+
+#include "can_driver.h"
+#define MAX_CAN_PAYLOAD_BTYES    8
+#define BUFF_SIZE 32
+
+#define WARN_OFFSET 0
+
+//MOTOR CONTROLLER
+#define MOTOR_CONTROLLER_1     0x10F8109A
+#define MOTOR_CONTROLLER_2     0x10F8108D
+//BMS
+#define BMS_TEMP             0x10
+#define BMS_WARN_OVERTEMP    0x06
```

Comment:
Keep naming consistent with only adding _WARN at the end of the non-warn name. Also, make the ID dependent on the WARN_OFFSET. See the sensor board example